enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/provider/ConsentType.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * PowerAuth Enrollment Server
+ * Copyright (C) 2022 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.wultra.app.onboardingserver.provider;
+
+/**
+ * Consent type.
+ *
+ * @author Lubos Racansky, lubos.racansky@wultra.com
+ */
+public enum ConsentType {
+
+    GDPR
```

Comment:
dtto - not sure if we can live with enum here...

Reply:
Changed to String.